packages/teams-js/src/private/teams.ts

```diff
@@ -82,4 +85,77 @@ export namespace teams {
   export function isSupported(): boolean {
     return runtime.supports.teams ? true : false;
   }
+
+  export namespace fullTrust {
+    export namespace joinedTeams {
+      /**
+       * @hidden
+       * Hide from docs
+       * ------
+       * Allows an app to retrieve information of all user joined teams
+       *
+       * @param teamInstanceParameters - OPTIONAL Flags that specify whether to scope call to favorite teams
+       * @returns Promise resolved containing information about the user joined teams or rejected with error
```

Comment:
Can you please update this to:
`@returns Promise that resolves with information about the user joined teams or rejects with an error when the operation has completed`